src/utils/src/ExponentialBackoffUtils.c

```diff
@@ -0,0 +1,223 @@
+#include "Include_i.h"
+
+//#define LOG_CLASS "XXXXX"
```

Comment:
remove?

Reply:
I think we use this class with other components. You can define your own log class for the exponential backoff or use the utils generic one

Reply:
Apparently we've already defined LOG_CLASS in PlatformUtils.h which indirectly gets included in this file

```
#ifndef LOG_CLASS
#define LOG_CLASS "platform-utils"
#endif
```

I'll just remove that line